examples/Basics/recommenders.py

```diff
@@ -116,10 +116,6 @@
 
 print(recommender)
 
-# Note that there are the additional keywords `hybrid_sampler` and `sampling_percentag`.
-# Their meaning and how to use and define it are explained in the hybrid backtesting example.
-# We thus refer to [`hybrid`](./../Backtesting/hybrid.md) for details on these.
```

Comment:
this is deletion of an essential piece of info
a simple solution would be to copy these isntructions into this suer guide here isntead of slashing them

Reply:
Agree. User Guide is probably the better place.

Reply:
then its OK for me